server/helpers/has-rich-article.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = exports = (prodList= []) => {
+	return (prodList.length >= 1) ? prodList.some(product => product.code === 'S2'): false;
+};
```

Comment:
My 2p, if this function is not used anywhere else then it doesn't need to be into a separate file itself.
In particular if it's such simple one line function.

Reply:
A note about `S2`, in master it's defined as a constant now:

https://github.com/Financial-Times/next-syndication-api/blob/master/config/default.yaml#L246

Reply:
@rosario just realised there is a bunch of changes in here that are no longer relevant. Hold fire on reviewing this I'm going to tidy up the PR